data/places/tei/4130.xml

```diff
@@ -143,6 +147,30 @@
             <ptr target="http://syriaca.org/cbss/GC2NFIZI"/>
             <citedRange unit="p">88-104</citedRange>
           </bibl>
+          <bibl xml:id="bib4130-4">
+            <title xml:lang="en" level="a">Gospel of Matthew</title>
+            <title xml:lang="en" level="m">The New Testament in Syriac</title>
+            <editor>G. H. Gwilliam</editor>
+            <editor>J. Pinkerton</editor>
+            <editor>John Gwynn</editor>
+            <ptr target="https://syriaccorpus.org/100#Head-id.3"/>
```

Comment:
We want to limit the `bibl/ptr/@target` to just be CBSS URIs, but this is a great idea for inclusion! We need to add this to our encoding Guidelines, but the way we'd do this would be to use the Digital Syriac Corpus' overall URI (http://syriaca.org/cbss/M9AGRZUE). Then you can add a cited range like below to add a hyperlink to a specific section of the text on the Corpus:
```
<citedRange unit="entry" target="https://syriaccorpus.org/100#Head-id.3">https://syriaccorpus.org/100#Head-id.3</citedRange>
```

Reply:
We hope eventually to have specific entries in CBSS for the texts on the Corpus, so you can cite each text specifically. But we haven't yet implemented that, so this is a valid workaround